app/controllers/welcome_controller.rb

```diff
@@ -0,0 +1,3 @@
+class WelcomeController < ApplicationController
```

Comment:
you never need to worry about a welcome page on an IC!

Reply:
manually write the path! No worries

Reply:
So does this pass?